### PR TITLE
Mc/uat batch 2

### PR DESCRIPTION
**Pull request by mcolella14**
Jira tickets:

- [DEVHUB-1133](https://jira.mongodb.org/browse/DEVHUB-1133)
- [DEVHUB-1158](https://jira.mongodb.org/browse/DEVHUB-1158)
- [DEVHUB-1181](https://jira.mongodb.org/browse/DEVHUB-1181)
- [DEVHUB-1182](https://jira.mongodb.org/browse/DEVHUB-1182)
- [DEVHUB-1189](https://jira.mongodb.org/browse/DEVHUB-1189)
- [DEVHUB-1199](https://jira.mongodb.org/browse/DEVHUB-1199)
- [DEVHUB-1205](https://jira.mongodb.org/browse/DEVHUB-1205)
- [DEVHUB-1161](https://jira.mongodb.org/browse/DEVHUB-1161)
- [DEVHUB-1190](https://jira.mongodb.org/browse/DEVHUB-1190)
- [DEVHUB-1154](https://jira.mongodb.org/browse/DEVHUB-1154)
- [DEVHUB-1197](https://jira.mongodb.org/browse/DEVHUB-1197)
- [DEVHUB-1200](https://jira.mongodb.org/browse/DEVHUB-1200)
- [DEVHUB-1202](https://jira.mongodb.org/browse/DEVHUB-1202)
- [DEVHUB-1203](https://jira.mongodb.org/browse/DEVHUB-1203)
- [DEVHUB-1206](https://jira.mongodb.org/browse/DEVHUB-1206)

### CODE REVIEW

File: src/components/search-filters/filter-group.tsx [hunk 1]
```diff
@@ -124,7 +124,8 @@ const FilterGroup: React.FunctionComponent<FilterGroupProps> = memo(
                                                     name={`${name} ${type} ${
                                                         isMobile ? 'mobile' : ''
                                                     }`}
-                                                    label={`${name} (${count})`}
+                                                    // label={`${name} (${count})`}
```
Comment (ajhenri) [inline src/components/search-filters/filter-group.tsx:127]: Is this comment needed?

File: src/components/search-filters/filter-group.tsx [hunk 2]
```diff
@@ -158,7 +159,8 @@ const FilterGroup: React.FunctionComponent<FilterGroupProps> = memo(
                                             name={`${name} ${type} ${
                                                 isMobile ? 'mobile' : ''
                                             }`}
-                                            label={`${name} (${count})`}
+                                            // label={`${name} (${count})`}
```
Comment (ajhenri) [inline src/components/search-filters/filter-group.tsx:162]: Same q as above

**Review by ajhenri — APPROVED** [review]
LGTM!

File: src/components/search-filters/filter-group.tsx (hunk 1 above)
Comment (mcolella14) [reply src/components/search-filters/filter-group.tsx:127]: So I think the plan is to have these counts added in a later version of the site, so I wanted to leave this here just as a reminder.

**Pull request merged**